### PR TITLE
[hi] Localize control-plane.md

### PR DESCRIPTION
**Pull request by bishal7679**
Signed-off-by: Bishal Das <bishalhnj127@gmail.com>

Localizes `content/en/docs/reference/glossary/control-plane.md` to Hindi `content/hi/docs/reference/glossary/control-plane.md`

Fixes :- #35600

### CODE REVIEW

File: content/hi/docs/reference/glossary/control-plane.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - fundamental
+---
+
+कंटेनर ऑर्केस्ट्रेशन लेयर जो कंटेनरों के जीवनचक्र को परिभाषित, परिनियोजित और प्रबंधित करने के लिए एपीआई और इंटरफेस को उजागर करता है।
+
+ <!--more-->
+
+यह लेयर कई अलग-अलग घटकों से बना है, जैसे (लेकिन इन तक सीमित नहीं):
+
+- {{< glossary_tooltip text="इटीसीडी" term_id="etcd" >}}
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/control-plane.md:20]:
```suggestion
- {{< glossary_tooltip text="etcd" term_id="etcd" >}}
```

File: content/hi/docs/reference/glossary/control-plane.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- {{< glossary_tooltip text="एपीआई सर्वर" term_id="kube-apiserver" >}}
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/control-plane.md:21]:
```suggestion
- {{< glossary_tooltip text="API सर्वर" term_id="kube-apiserver" >}}
```

File: content/hi/docs/reference/glossary/control-plane.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- {{< glossary_tooltip text="सचेंडूलर" term_id="kube-scheduler" >}}
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/control-plane.md:22]: `शेड्यूलर`  resembles the `scheduler` more phonetically.
```suggestion
- {{< glossary_tooltip text="शेड्यूलर" term_id="kube-scheduler" >}}
```

File: content/hi/docs/reference/glossary/control-plane.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+title: कण्ट्रोल प्लेन (Control Plane)
+id: control-plane
+date: 2019-05-12
+full_link:
+short_description: >
+  कंटेनर ऑर्केस्ट्रेशन लेयर जो कंटेनरों के जीवनचक्र को परिभाषित, परिनियोजित और प्रबंधित करने के लिए एपीआई और इंटरफेस को उजागर करता है।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/control-plane.md:7]: Just a small nit:
```suggestion
  कंटेनर ऑर्केस्ट्रेशन लेयर जो कंटेनरों के जीवनचक्र को परिभाषित, परिनियोजित और प्रबंधित करने के लिए API और इंटरफेस को उजागर करता है।
```

**Comment by bishal7679** [comment]
@Babapool Nice suggestions
But Can you tell me how to do that?
like how you added commit in my PR as a suggestion?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

File: content/hi/docs/reference/glossary/control-plane.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+---
+title: कण्ट्रोल प्लेन (Control Plane)
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/control-plane.md:2]:
```suggestion
title: कंट्रोल प्लेन (Control Plane)
```

File: content/hi/docs/reference/glossary/control-plane.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+यह लेयर कई अलग-अलग घटकों से बना है, जैसे (लेकिन इन तक सीमित नहीं):
+
+- {{< glossary_tooltip text="etcd" term_id="etcd" >}}
+- {{< glossary_tooltip text="API सर्वर" term_id="kube-apiserver" >}}
+- {{< glossary_tooltip text="शेड्यूलर" term_id="kube-scheduler" >}}
+- {{< glossary_tooltip text="कंट्रोलर मैनेजर" term_id="kube-controller-manager" >}}
+- {{< glossary_tooltip text="क्लाउड कंट्रोलर मैनेजर" term_id="cloud-controller-manager" >}}
+
+इन घटकों को पारंपरिक ऑपरेटिंग सिस्टम सेवाओं (डेमॉन) या कंटेनर के रूप में चलाया जा सकता है। इन घटकों को चलाने वाले मेजबानों को ऐतिहासिक रूप से {{< glossary_tooltip text="मास्टरस" term_id="master" >}} कहा जाता था।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/control-plane.md:26]:
```suggestion
इन घटकों को पारंपरिक ऑपरेटिंग सिस्टम सेवाओं (डैमन) या कंटेनर के रूप में चलाया जा सकता है। इन घटकों को चलाने वाले मेजबानों को ऐतिहासिक रूप से {{< glossary_tooltip text="मास्टर्स" term_id="master" >}} कहा जाता था।
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

File: content/hi/docs/reference/glossary/control-plane.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+कंटेनर ऑर्केस्ट्रेशन लेयर जो कंटेनरों के जीवनचक्र को परिभाषित, परिनियोजित और प्रबंधित करने के लिए एपीआई और इंटरफेस को उजागर करता है।
+
+ <!--more-->
+
+यह लेयर कई अलग-अलग घटकों से बना है, जैसे (लेकिन इन तक सीमित नहीं):
+
+- {{< glossary_tooltip text="etcd" term_id="etcd" >}}
+- {{< glossary_tooltip text="API सर्वर" term_id="kube-apiserver" >}}
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/control-plane.md:21]:
```suggestion
- {{< glossary_tooltip text="क्यूब-API सर्वर" term_id="kube-apiserver" >}}
```

File: content/hi/docs/reference/glossary/control-plane.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+- {{< glossary_tooltip text="शेड्यूलर" term_id="kube-scheduler" >}}
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/control-plane.md:22]:
```suggestion
- {{< glossary_tooltip text="क्यूब-शेड्यूलर" term_id="kube-scheduler" >}}
```

**Review by divya-mohan0209 — DISMISSED** [review]
Additional set of nits

**Comment by divya-mohan0209** [comment]
@bishal7679 : Please retain only one PR among #36121 & aggregate all the suggestions in one of them.

**Comment by divya-mohan0209** [comment]
@bishal7679 Any update on this?

File: content/hi/docs/reference/glossary/control-plane.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+- {{< glossary_tooltip text="कंट्रोलर मैनेजर" term_id="kube-controller-manager" >}}
+- {{< glossary_tooltip text="क्लाउड कंट्रोलर मैनेजर" term_id="cloud-controller-manager" >}}
+
+इन घटकों को पारंपरिक ऑपरेटिंग सिस्टम सेवाओं (डैमन) या कंटेनर के रूप में चलाया जा सकता है। इन घटकों को चलाने वाले मेजबानों को ऐतिहासिक रूप से {{< glossary_tooltip text="मास्टर्स" term_id="master" >}} कहा जाता था।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/control-plane.md:26]:
```suggestion
इन घटकों को पारंपरिक ऑपरेटिंग सिस्टम सेवाओं (डेमॉन) या कंटेनर के रूप में चलाया जा सकता है। इन घटकों को चलाने वाले मेजबानों को ऐतिहासिक रूप से {{< glossary_tooltip text="मास्टर्स" term_id="master" >}} कहा जाता था।
```

**Review by divya-mohan0209 — COMMENTED** [review]
Small nit. Otherwise LGTM.

**Comment by Babapool** [comment]
@bishal7679 can you please squash your commits to make the PR ready to merge. Thanks.

**Review by divya-mohan0209 — COMMENTED** [review]
@Babapool : Please can you LGTM it if everything looks okay? I've added the squash commits label since this has been pending for quite some time now.

/approve

**Review by Babapool — COMMENTED** [review]
Thanks!
/lgtm

**Pull request merged**